index.js

```diff
@@ -2420,7 +2438,7 @@
   }
   _.encase = {
     consts: {},
-    types: [$.Fn (a) (b), a, $.Either ($.Error) (b)],
+    types: [Throwing (e) (a) (b), a, $.Either (e) (b)],
```

Comment:
Oh! I just found the answer to my own question! It's the change from `$.Error` to `e` in the return type.